BookingsApi/BookingsApi.DAL/Commands/RemoveParticipantFromHearingCommand.cs

```diff
@@ -35,11 +35,10 @@ public async Task Handle(RemoveParticipantFromHearingCommand command)
                 .SingleOrDefaultAsync(x => x.Id == command.HearingId);
             
             if (hearing == null)
-            {
                 throw new HearingNotFoundException(command.HearingId);
```

Comment:
we should probably put styling rules in the repo to keep it consistent between devs and IDEs

Reply:
Yeah, personally I think one-line if statments should take advantage of not needing braces to save space and make the code look tidier.